imports/plugins/core/orders/client/containers/ordersListContainer.js

```diff
@@ -28,26 +26,40 @@ class OrdersListContainer extends Component {
     this.state = {
       selectedItems: [],
       orders: props.orders,
-      hasMoreOrders: props.hasMoreOrders(),
-      multipleSelect: false
+      multipleSelect: false,
+      shipping: {
+        picked: false,
+        packed: false,
+        labeled: false,
+        shipped: false
+      },
+      renderFlowList: false,
+      isLoading: {
+        picked: false,
+        packed: false,
+        labeled: false,
+        shipped: false
+      }
     };
-
-    this.handleClick = this.handleClick.bind(this);
-    this.handleDisplayMedia = this.handleDisplayMedia.bind(this);
-    this.handleSelect = this.handleSelect.bind(this);
-    this.selectAllOrders = this.selectAllOrders.bind(this);
   }
 
   componentWillReceiveProps = (nextProps) => {
     this.setState({
-      orders: nextProps.orders,
-      hasMoreOrders: nextProps.hasMoreOrders()
+      orders: nextProps.orders
     });
   }
 
   handleSelect = (event, isInputChecked, name) => {
     this.setState({
-      multipleSelect: false
+      multipleSelect: false,
+      renderFlowList: false
+    });
+    shippingStrings.forEach((string) => {
```

Comment:
Not sure settings the variable name to be the name of a type is probably a good idea.